.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+	<extension>
+		<groupId>org.eclipse.tycho</groupId>
+		<artifactId>tycho-build</artifactId>
+		<version>4.0.1</version>
```

Comment:
@akurtakov in eclipse.platform.releng.aggregator\.mvn\extensions.xml is a 4.0.4 what is the right value here?

Reply:
I copied from platform.ui... so if 4.0.4 platform.ui should also be updated